boggle.js

```diff
@@ -38,39 +29,31 @@ function removeElement(selectedDice, index) {
   return item;
 }
 
-function shuffle(selectedDice) {
+function shuffle(dice) {
   while (dice.length > 0) {
-  // console.log(dice);
     let randIndex = randomRange(dice.length);
     let removedElem = removeElement(dice, randIndex);
-    shuffledDice.push(removedElem);
-  // return shuffledArray;
-    // console.log(removedElem);
+    shuffledDice.push(`${removedElem} `);
   }
   return shuffledDice;
 }
 
 function shake(board) {
-  let finalDice = shuffle(dice);
-  // console.log(finalArray);
+  let shuffledDice = shuffle(dice);
   let diceIndex = 0;
-  for (let a = 0; a < 4; a++) {
-    for (let b = 0; b < 4; b++) {
-      // let randomDie = allArrays[Math.floor(Math.random() * 16)];
-      let randomChar = finalDice[diceIndex].charAt(Math.floor(Math.random() * 6));
-      // console.log('Die, char:', randomChar, allArraysIndex);
+  for (let a = 0; a < board.length; a++) {
+    for (let b = 0; b < board[0].length; b++) {
```

Comment:
Careful. While this works in this case, if one of the rows had a different number of columns, this code would break. You want `b < board[a].length`.

Also, just a note on convention, typically the variables `i`, `j`, and `k` are used to signify indices. You can also use more expressive variable names, like `row` and `col`.